site/topics/transistors-gates/transistors-gates.rst

```diff
@@ -11,6 +11,94 @@ Transistors and Logic Gates
 Symbols
 =======
 
+* For this course, software called `Digital <https://github.com/hneemann/Digital>`_ will be used to create digital circuits
+* There are a number of symbols representing components that will need to be understood
+* Although some of you may be familiar with electronic component symbols, there are some things to note
+
+    * Certain components have different symbols depending on their context
+    * Some components have different symbols in different regions of the world
+
+
+* When using Digital
+
+    * An up arrow is used to signify a positive charge
+    * A single horizontal line represents ground --- something with no charge; neutral
+    * Typically, within the context of computer architecture, the voltage between this positive charge and ground is 5V
+
+
+.. figure:: vcc_ground.png
+    :width: 50 px
+    :align: center
+
+    Screenshot from Digital of a positive charge (up arrow) connected directly to ground (horizontal line).
+
+
+* A single horizontal line is common within computer architecture and is the symbol found in Digital
+
+    * There are other common symbols for ground depending on the context of how it is used
+
+
+.. figure:: ground_symbols.png
+    :width: 333 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Ground_(electricity)
+
+    Examples of three commonly used ground symbols. These have slightly different meanings, but all ultimately signify
+    something with a neutral charge.
+
+
+* In the above example, a voltage source was connected directly to ground
+* In practice, this is a bad idea as that could cause damage and injury
+
+    * Current would be too high
+
+
+* A resistor, for example, could be added to the circuit to limit its current
+* Below is an example of the simple circuit containing a resistor
+
+.. figure:: vcc_resistor_ground.png
+    :width: 50 px
+    :align: center
+
+    Screenshot from Digital of a positive charge (up arrow) connected to ground (horizontal line) through a resistor
+    (box).
+
+
+* In North America, a sawtooth symbol is used to represent a resistor, but elsewhere in the world it is typically a box
+
+    * Like in the above figure
```

Comment:
neat. til

Reply:
There's more nuance on the standards, but this is good enough.